test/dx_mgn_pool.js

```diff
@@ -66,6 +66,27 @@ contract("DxMgnPool", (accounts) => {
   })
 
   describe("participateInAuction()", () => {
+
+    it("non-owner can't ping it", async() => {
+      const dx = await DutchExchange.new()
+
+      const depositTokenMock = await MockContract.new()
+      const secondaryTokenMock = await MockContract.new()
+      const mgnTokenMock = await MockContract.new()
+      const dxMock = await MockContract.new()
+      const poolingEndBlock = (await web3.eth.getBlockNumber()) + 100
+      const instance = await DxMgnPool.new(depositTokenMock.address, secondaryTokenMock.address, mgnTokenMock.address, dxMock.address, poolingEndBlock)
+      
+      await depositTokenMock.givenAnyReturnBool(true)
+      
+      await dxMock.givenAnyReturnUint(42)
```

Comment:
are the dx mock expectations needed?

Reply:
yes, for the getAuctionIndex in particpateInAuction